AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -54,6 +54,8 @@ armidaleexpress.com.au,bendigoadvertiser.com.au,canberratimes.com.au,centralwest
 !
 ! SECTION: Subscriptions - Regular rules
 !
+vox.com##li:has(> a[href="/newsletters"])
```

Comment:
Not a popup. User has to press on Menu to see it.
Imho move to Annoyances -> Other -> Annoyances.

Else pull request looks fine to me. Unless colleagues disagree about rule move.

Reply:
![image](https://github.com/AdguardTeam/AdguardFilters/assets/8361299/0b7e44e0-f559-484e-a47d-87c15088ade7)
It should not be blocked at all.

Reply:
```suggestion
```